### PR TITLE
Changed the frozen amount to be at least 40,000 BOA

### PR DESCRIPTION
**Pull request by TrustHenry**
according to the Yellow White Paper
> It has at least 40,000 coins

### CODE REVIEW

File: source/agora/consensus/Validation.d [hunk 1]
```diff
@@ -468,8 +468,8 @@ unittest
         );
         secondTx.inputs[0].signature = key_pairs[0].secret.sign(hashFull(secondTx)[]);
 
-        // Second Transaction is invalid.
-        assert(!secondTx.isValid(findUTXO, 0));
+        // Second Transaction is valid.
+        assert(secondTx.isValid(findUTXO, 0));
```
Comment (AndrejMitrovic) [inline source/agora/consensus/Validation.d:472]: Look at the comment in line 446. That comment needs to be changed too.

File: source/agora/consensus/Validation.d [hunk 2]
```diff
@@ -94,8 +94,8 @@ public string isInvalidReason (const Transaction tx, UTXOFinder findUTXO,
                 return "Transaction: Can only freeze a Payment transaction";
         }
 
-        if (sum_unspent != Amount.FreezeAmount)
-            return "Transaction: Only available when the amount is 40,000";
+        if (sum_unspent.integral() < Amount.FreezeAmount.integral())
+            return "Transaction: available when the amount is least 40,000 BOA";
```
Comment (AndrejMitrovic) [inline source/agora/consensus/Validation.d]:
```suggestion
            return "Transaction: available when the amount is at least 40,000 BOA";
```

File: source/agora/consensus/Validation.d [hunk 3]
```diff
@@ -94,8 +94,8 @@ public string isInvalidReason (const Transaction tx, UTXOFinder findUTXO,
                 return "Transaction: Can only freeze a Payment transaction";
         }
 
-        if (sum_unspent != Amount.FreezeAmount)
```
Comment (AndrejMitrovic) [inline source/agora/consensus/Validation.d:97]: `Amount.FreezeAmount` should then be renamed to `Amount.MinFreezeAmount`, to make it obvious that it's the minimum freeze amount, not exact freeze amount.

Comment (TrustHenry) [reply source/agora/consensus/Validation.d:97]: applied the suggest

File: source/agora/consensus/Validation.d (hunk 2 above)
Comment (TrustHenry) [reply source/agora/consensus/Validation.d]: applied the suggest

File: source/agora/consensus/Validation.d (hunk 1 above)
Comment (TrustHenry) [reply source/agora/consensus/Validation.d:472]: Good eye :) applied the suggest

**Review by linked0 — APPROVED** [review]

**Review by AndrejMitrovic — APPROVED** [review]

**Comment by Geod24** [comment]
@TrustHenry : Please leave an empty line between the title and the commit message.
Currently your title is over 100 character, which makes it very hard to read.
This is what I see in the console:
Screen Shot 2019-12-16 at 18 26 02

**Pull request merged**